src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairHMM.java

```diff
@@ -101,12 +101,16 @@ public VectorLoglessPairHMM() throws UserException.HardwareFeatureException {
                     logger.info("libVectorLoglessPairHMM found in JVM library path");
                 } catch (UnsatisfiedLinkError e) {
```

Comment:
Wish there were a way to unit test the two failure modes here (AVX not supported and library not loadable) via mocks or something similar